addons/web/static/src/js/views/calendar/calendar_renderer.js

```diff
@@ -562,8 +562,8 @@ return AbstractRenderer.extend({
     },
     /**
      * Add attribute to the element
-     * @param {jQueryElement} element 
-     * @param {Object} event 
+     * @param {jQueryElement} element
+     * @param {Object} event
```

Comment:
This introduces a useless diff don't you think?

Reply:
Oups, yeah it is my editor, sorry, I will remove it